packages/nouns-contracts/contracts/governance/data/NounsDAOData.sol

```diff
@@ -174,6 +185,10 @@ contract NounsDAOData is OwnableUpgradeable, UUPSUpgradeable, NounsDAODataEvents
             encodedProp = abi.encodePacked(proposalIdToUpdate, encodedProp);
         }
 
+        if (msg.value > 0) {
```

Comment:
suggestion: check that `feeRecipient` is not zero?
if `feeRecipient` is zero, then this contract can accumulate eth and the owner can withdraw later